### PR TITLE
Branch OV - Sort by parents instead of time + PP #577

### PR DESCRIPTION
**Pull request by bigld**

### CODE REVIEW

**Comment by bigld** [comment]
for testing you can use https://github.com/bigld/abapgit_dbtests_TABL.git and https://github.com/bigld/abapgit_dbtests.git

there a commits with changed timestamps and some to test the compress function

**Comment by larshp** [comment]
#577

File: src/zcl_abapgit_branch_overview.clas.abap [hunk 1]
```diff
@@ -54,12 +55,18 @@ CLASS zcl_abapgit_branch_overview DEFINITION
     METHODS determine_tags
       RAISING
         zcx_abapgit_exception .
-ENDCLASS.
+    METHODS _sort_commits CHANGING ct_commits TYPE ty_commits.
+    METHODS _get_1st_child_commit
+      IMPORTING itr_commit_sha1s    TYPE tyt_commit_sha1_range
```
Comment (larshp) [inline src/zcl_abapgit_branch_overview.clas.abap]: it_ prefix is what is used in other places, but anyhow, should add it in http://docs.abapgit.org/development/guidelines.html

**Comment by larshp** [comment]
I'm a bit concerned about performance but lets just try this, merging, thanks 👍

**Pull request merged**